Readme.md

```diff
@@ -0,0 +1,7 @@
+This is a Go package for manipulating paragraphs of text.
```

Comment:
I prefer to do one complete thing at a time in each PR. Renaming & updating the readme seems like a separate thing from updating dependencies to me. What do you think?